core/src/main/java/org/geysermc/geyser/GeyserBootstrap.java

```diff
@@ -68,11 +69,14 @@
     GeyserLogger getGeyserLogger();
 
     /**
-     * Returns the current CommandManager
+     * Creates a command manager for {@link GeyserImpl} to use.
      *
-     * @return The current CommandManager
+     * @return a new CommandManager instance
      */
-    CommandManager getGeyserCommandManager();
+    @Contract("_ -> new")
+    default CommandManager createGeyserCommandManager(GeyserImpl geyser) {
+        return new CommandManager(geyser);
```

Comment:
Just a heads up that in my Sponge PR I'll have to change this javadoc to "allow" platforms to provide an existing command manager, since the sponge command manager needs to be created before the connector. There are some other things on my mind as well but they're manageable. 

Also the _ in the contract infers null values are allowed as per jetbrains docs

Reply:
Fun.